### ISSUE TITLE
Add way to set document root for file server

### ISSUE THREAD

**Issue by ghost** [w3c/wot-go#2]
Currently we do not have a way to load configuration, so I left the file server loading files from a relative `www` directory.

**Comment by draggett** [comment]
> On 27 Jun 2015, at 14:31, Herbert Fischer notifications@github.com wrote:
> 
> Currently we do not have a way to load configuration, so I left the file server loading files from a relative www directory.

Good point.  For the NodeJS server I introduced a config.js file with a view to supporting rich config settings.  We have the path for the for the HTTP root folder. We also need to define the base URI for thing models registered for new things. Please feel free to create a way to manage the configuration settings.

—
   Dave Raggett <dsr@w3.org <mailto:dsr@w3.org>>

**Comment by ghost** [comment]
Ok. I will.
